LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Amadeus s.a.s
+   Copyright [yyyy] The Perses Authors
```

Comment:
I'm not sure that the LICENSE itself should mention the copyright owners at all, as these are just general instructions on how to apply the Apache 2 license. See also: https://github.com/prometheus/prometheus/blob/ee7e0071d10e795cb78a8e50764e45cbcaf8c29a/LICENSE#L189